lib/instance-detail-helper.js

```diff
@@ -4,6 +4,7 @@ const toNS = require('mongodb-ns');
 const security = require('mongodb-security');
 const ReadPreference = require('mongodb').ReadPreference;
 const URL = require('mongodb-url');
+const getMongoDBBuildInfo = require('mongodb-build-info');
```

Comment:
If you're bringing in this package already, it'd be good to use its `isGenuineMongoDB` function and remove its duplicate from here.

Reply:
you can also use `mongodb-build-info` to do the check for enterprise_module by using the `isEnterprise()` helper.